test/tstHeapOperations.cpp

```diff
@@ -173,6 +173,18 @@ BOOST_AUTO_TEST_CASE(sort_heap)
   }
 }
 
+BOOST_AUTO_TEST_CASE(heap_sort)
+{
+  for (auto heap :
```

Comment:
Do not call it heap as it will create confusion.  Call it `v` or whatever you like.  This is not a heap, this is an unordered array/vector.

Reply:
Fixed.